query-engine/connector-test-kit-rs/query-engine-tests/tests/raw/sql/casts.rs

```diff
@@ -5,7 +5,20 @@ use query_engine_tests::*;
 mod casts {
     use query_engine_tests::{fmt_query_raw, run_query, RawParam};
 
-    #[connector_test]
+    // The following tests are excluded for driver adapters. The underlying
+    // driver rejects queries where the values of the positional arguments do
+    // not match the expected types. As an example, the following query to the
+    // driver
```

Comment:
Does that mean that our raw query API with driver adapters is less accepting than our Rust one?

(Not a problem at all, just interesting)

Reply:
Yes, it's a little bit more strict.